tasks/server_keys.yml

```diff
@@ -91,3 +97,37 @@
     chdir: "{{openvpn_key_dir}}"
     creates: dh.pem
   when: not (openvpn_use_pregenerated_dh_params|bool)
+
+- name: install ca.conf config file
+  template:
+    src: ca.conf.j2
+    dest: "{{openvpn_key_dir}}/ca.conf"
+    owner: root
+    group: root
+    mode: 744
+
+- name: create initial certificate revocation list squence number
+  command: echo 00 > crl_number
+    creates: crl_number
```

Comment:
This is failing the syntax checks, I'm pretty sure you're missing the `args:` before `creates:`. (Also for that matter, you might want `chdir` as well.)

While looking for details on what's wrong, I also found [this](https://docs.ansible.com/ansible/latest/command_module.html#notes): 
> If you want to run a command through the shell (say you are using <, >, |, etc), you actually want the shell module instead. The command module is much more secure as it’s not affected by the user’s environment.
So you probably want to switch to `shell` instead of `command`.